disttask/loaddata/dispatcher.go

```diff
@@ -147,13 +175,23 @@ func (h *flowHandle) switchTiKV2NormalMode(ctx context.Context, logger *zap.Logg
 	h.lastSwitchTime.Store(time.Time{})
 }
 
-// postProcess does the post-processing for the task.
-func postProcess(ctx context.Context, handle dispatcher.TaskHandle, gTask *proto.Task, logger *zap.Logger) error {
-	taskMeta := &TaskMeta{}
-	err := json.Unmarshal(gTask.Meta, taskMeta)
-	if err != nil {
+// preProcess does the pre processing for the task.
+func preProcess(ctx context.Context, handle dispatcher.TaskHandle, gTask *proto.Task, taskMeta *TaskMeta, logger *zap.Logger) error {
+	logger.Info("pre process", zap.Any("task_meta", taskMeta))
```

Comment:
zap.Any("task_meta", taskMeta) is not a good choice. Please log what you want to log.